core/viewer/src/main/resources/resources/templates/PDFJSViewer.gtmpl

```diff
@@ -52,7 +52,7 @@ if (maximumPage > 0 && maximumPage <= pdfViewerService.getMaxPages() && fileSize
         <div id="sidebarContainer">
             <div id="toolbarSidebar">
                 <div class="splitToolbarButton toggled btn-group">
-                    <button id="viewThumbnail" class="btn hidden" data-placement="right" rel="tooltip" data-original-title="Show Thumbnails" tabindex="2" data-l10n-id="thumbs">
+                    <button id="viewThumbnail" class="btn hidden" data-placement="right" rel="tooltip" title="Show Thumbnails" tabindex="2" data-l10n-id="thumbs">
```

Comment:
Can you explain this change ?

Reply:
the `moz10l` translate only the `title` attribute

Reply:
do we need to keep data-original-title in addition to title ?

Reply:
If we keep it, two tool-tips will be displayed, one always in English and the second translated

Reply:
ok